README.md

```diff
@@ -5,3 +5,17 @@ This repository will (eventually) contain all the scripts that are pieced togeth
 ## Integrations
 
 Under the integrations folder, you'll find the scripts that get executed when a particular integration gets used in a build.
+
+#### Contribute
+- create following file structure inside `integrations` directory
+```
+integrationName/
+`-- cli
+    |-- cleanup.sh
+    `-- init.sh
+```
+- Write `init.sh` and `cleanup.sh`
```

Comment:
Lets expand on this:
- Populate `init.sh` with the script required to configure the integration.
- Populate `cleanup.sh` with the script required to clean up the integration.
- Use existing integration handler as references.